lib/rabbit/event_handler.rb

```diff
@@ -7,7 +7,8 @@ class Rabbit::EventHandler
   include Tainbox
 
   attribute :project_id
-  attr_accessor :data
+  attr_accessor :data, :message_info
```

Comment:
it looks a little messy to use in the same class both tainbox attribute & attr_accessor

Reply:
fixed